### PR TITLE
{tools}[foss+intel/2015a] bibtexparser 0.5 + 0.6.0 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/b/bibtexparser/bibtexparser-0.5-intel-2015a-Python-2.7.9.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+easyblock = 'PythonPackage'
+
+name = 'bibtexparser'
+version = '0.5'
```
Comment (wpoely86) [inline easybuild/easyconfigs/b/bibtexparser/bibtexparser-0.5-intel-2015a-Python-2.7.9.eb]: why two versions? And why `0.6.0` vs `0.5`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux RHEL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/47233387bcc29a34e211 for a full test report.

File: easybuild/easyconfigs/b/bibtexparser/bibtexparser-0.5-intel-2015a-Python-2.7.9.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/b/bibtexparser/bibtexparser-0.5-intel-2015a-Python-2.7.9.eb]: 0.6.0 is the most recent one, but I need 0.5 as a dependency for something

I didn't invent this crazy versioning scheme; 0.5.0 doesn't exist :)

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**